src/routes/accounts.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+"""Definitions for the routes of the accounts API."""
+from connexion.lifecycle import ConnexionResponse
+
+from src.schemas.account_schema import AccountsResponse, _AccountSchema
+from src.models.account_model import AccountModel
+
+
+def get_accounts(offset=0, record_count=1000):
+    """Return a page from the list of all accounts that are present in the system."""
+    if record_count > 1000 or offset < 0:
+        return ConnexionResponse(status_code=400)
```

Comment:
Rather than giving Bad request, its good if we can add descriptive message also, can we do this in ConnexionResponse?

Reply:
Yes we can, will do